browser/main/lib/RcParser.js

```diff
@@ -0,0 +1,23 @@
+import _ from 'lodash'
+
+const path = require('path')
+const sander = require('sander')
+
+function parse (boostnotercPath) {
```

Comment:
Is it necessary that receiving `boostnotercPath` from outside?
I think the path of `.boostnoterc` is not multiple. So declare the path on this file is better.